docs/UserGuide.md

```diff
@@ -204,8 +204,11 @@ Lists all existing medications in the inventory.
 * All parameters for `liststock` command are optional, you can choose to list medication by any of the parameters.
 * You are able to `liststock` by any column and sort or reverse sort them.
 * When you update an order information, MediVault reflects the pending stocks shown here.
+* The total pending quantity will be shown if there are orders for a medication.
+* `low/LESS_THAN_OR_EQUAL_QUANTITY` and `expiring/LESS_THAN_OR_EQUAL_EXPIRY_DATE` parameters can be used to search for
+stocks with low quantity and expiring stocks. 
 
-Format: `liststock {i/STOCK_ID n/NAME p/PRICE q/QUANTITY e/EXPIRY_DATE d/DESCRIPTION m/MAX_QUANTITY sort/COLUMN_NAME rsort/COLUMN_NAME}`
+Format: `liststock {i/ID p/PRICE q/QUANTITY low/LESS_THAN_OR_EQUAL_QUANTITY e/EXPIRY_DATE expiring/LESS_THAN_OR_EQUAL_EXPIRY_DATE d/DESCRIPTION m/MAX_QUANTITY sort/COLUMN_NAME rsort/COLUMN NAME}`
```

Comment:
Should be `COLUMN_NAME` for rsort instead?

Reply:
Noted, will update it